cmd/admin-server-info.go

```diff
@@ -90,6 +90,49 @@ func getLocalCPULoad(endpointZones EndpointZones, r *http.Request) ServerCPULoad
 	}
 }
 
+func getLocalDrivesOBD(endpointZones EndpointZones, r *http.Request) madmin.ServerDrivesOBDInfo {
+	var drivesOBDInfo []madmin.DriveOBDInfo
+	for _, ep := range endpointZones {
+		for _, endpoint := range ep.Endpoints {
+			// Only proceed for local endpoints
```

Comment:
```suggestion
                      if !endpoint.IsLocal {
                         continue
                      }
```
and remove the `if endpoint.IsLocal {` to avoid nesting